### PR TITLE
Rails 6.0.3.1 with Webpacker upgrade

### PR DESCRIPTION
**Pull request by vladimirbazhanov**
# Summary

- Rails version upgraded to 6.0.3.1
- Added Webpacker as default JavaScript compiler
- Updated linters (Rubocop, slim-lint and others"
- Ruby version used 2.7.1
- jQuery updated to 3rd version and connected using Webpacker
- jQuery UJS removed and replaced with @rails/ujs loaded with Webpacker
- Removed support for CoffeeScript
- bundler-audit uncommented and all security issues fixed

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,20 +1,22 @@
 source "https://rubygems.org"
 
-ruby "2.5.7"
+ruby "2.7.1"
 
 gem "pg"
-gem "rails", "5.2.4.3"
+
+# gem "rails", "5.2.4.3"
```
Comment (DmitryBarskov) [inline Gemfile:7]:
```suggestion
```

File: app/javascript/packs/hello_react.jsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
+// Run this example by adding <%= javascript_pack_tag 'hello_react' %> to the head of your layout file,
```
Comment (ArthurZaharov) [inline app/javascript/packs/hello_react.jsx:1]: Do we need this file?

Comment (vladimirbazhanov) [reply app/javascript/packs/hello_react.jsx:1]: This is example file generated when you create new app or initialize webpacker. When I made Rails upgrade I was trying to leave things as they generated by default to make minimum of changes.

Sure, we can remove this or even make some other 'demo' component, but I think in "base" project it's better to leave it untouched.

File: .rubocop.yml [hunk 3]
```diff
@@ -1,35 +1,63 @@
...
     - tmp/**/*
 
+Capybara/FeatureMethods:
+  EnabledMethods:
+    - background
+    - feature
+    - scenario
+
+# Rails
+
```
Comment (timurvafin) [inline .rubocop.yml:25]: Empty line?

File: .rubocop.yml [hunk 4]
```diff
@@ -1,35 +1,63 @@
...
+    - production
+
 Rails/HasAndBelongsToMany:
   Enabled: false
 
 Rails/OutputSafety:
   Enabled: false
 
+#RSpec
+
```
Comment (timurvafin) [inline .rubocop.yml:43]: Empty line?

File: .rubocop.yml [hunk 5]
```diff
@@ -1,35 +1,63 @@
...
+    - staging
+    - production
+
 Rails/HasAndBelongsToMany:
   Enabled: false
 
 Rails/OutputSafety:
   Enabled: false
 
+#RSpec
```
Comment (timurvafin) [inline .rubocop.yml:42]: Space between `#` and `Rspec`

**Pull request merged**